Add changelog for 1.31.2 patch release (#655)
conduktor/conduktor-docs: src/pages/changelog/Console-1.31.2.md

---
date: 2025-03-03
title: Hotfix for Console 1.31.2
description: docker pull conduktor/conduktor-console:1.31.2
solutions: console
tags: fix
---

*Release date: {frontMatter.date.toISOString().slice(0, 10)}*

### Fixes
- Fixed prefixed ACLs not displaying correctly in the Service Account UI

> - Fixed dangling Service Account metadata incorrectly appearing in the UI

Reviewer: Can you add some more detail to these descriptions?  Put yourself in the shoes of the user; would you understand what had been fixed from these descriptions?

Author: I've reworded the dangling metadata sentence to be clearer. Don't think it needs more detail than that as it seems self descriptive to me now

- Fixed an issue allowing creation of Service Accounts without ACLs

### Known Issues

> - When removing ACLs in the Service Account UI, you can not remove the last ACL

Reviewer: ```suggestion
- When removing ACLs in the Service Account UI, you cannot remove the last ACL
```

Is there a workaround for this?  If so, it should be mentioned.  If not, do we have a plan to fix this (i.e. in the next release)?

Author: I've added the workaround that Julien mentioned in our earlier discussion and mentioned that we will address this in the next release 👍

---
date: 2025-03-03
title: Hotfix for Console 1.31.2
description: docker pull conduktor/conduktor-console:1.31.2
solutions: console
tags: fix
---

*Release date: {frontMatter.date.toISOString().slice(0, 10)}*

### Fixes
- Fixed prefixed ACLs not displaying correctly in the Service Account UI
- Fixed an issue where Service Accounts with no ACL were incorrectly shown in the UI
- Fixed an issue allowing creation of Service Accounts without ACLs

### Known Issues
- When removing ACLs in the Service Account UI, you cannot remove the last ACL

>  - As a workaround, you can create remove that last ACL, then create a new ACL against a resource name that does not exist.

Reviewer: Typo?  `create remove that last ACL...`
